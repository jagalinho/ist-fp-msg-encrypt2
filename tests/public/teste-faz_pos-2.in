pos = faz_pos(0,8)
pos
