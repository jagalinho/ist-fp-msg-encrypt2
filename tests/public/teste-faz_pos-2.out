faz_pos(5-5,4+4)
